server.js

```diff
@@ -85,21 +85,28 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.get(pathPrefix + 'login', login.get);
 app.post(pathPrefix + 'login', jsonParser, login.post);
 
-app.all(appPrefix + 'update_settings/*', function(req, res) {
-  // don't audit the app settings
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_revs_diff', function(req, res) {
-  // don't audit the _revs_diff
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_local/*', function(req, res) {
-  // don't audit the _local docs
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_bulk_get/*', function(req, res) {
-  // don't audit the _bulk_get request
-  proxy.web(req, res);
+// Requests we don't wish to audit
+// (usually because POST is just used here to pass more parameters)
+[
+  appPrefix + 'update_settings/*',
+  // Replication machinery we don't care to audit
+  pathPrefix + '_local/*',
+  pathPrefix + '_revs_diff',
+  pathPrefix + '_missing_revs',
+  // Can use POST for specifiying ids
+  pathPrefix + '_bulk_get',
+  pathPrefix + '_all_docs',
+  pathPrefix + '_design/*/_list/**',
+  pathPrefix + '_design/*/_show/**',
+  pathPrefix + '_design/*/_view/*',
+  pathPrefix + '_changes',
+  // Interacting with mongo filters uses POST
+  pathPrefix + '_find',
+  pathPrefix + '_explain'
+].forEach(function(url) {
```

Comment:
I think it'd be clearer to pull this array out as a constant, something like "UNAUDITED_ENDPOINTS", idk. Then while iterating over them add the `pathPrefix`.